index.html

```diff
@@ -259,6 +156,74 @@ <h2 class="contact--me">Contact me</h2>
       <button type="submit" class="see--project contact-button">Get in touch</button>
     </form>
   </section>
+  <section class="pop-up" id="modal">
+    <div class="overlay">
+      <div class="head">
+        <h2>Tonic</h2>
+        <img src="images/pop-up-close-icon.png" class="pop-up-close-icon" alt="">
+      </div>
+      <div class="project-description">
+        <ul class="experience-list">
+          <li>CANOPY</li>
+          <li>Backend Dev</li>
+          <li>2015</li>
+        </ul>
+        <div class="detail-img-desk">
+        </div>
+        <div class="detail-img">
+        </div>
+      </div>
+      <div class="flex-pop">
+        <div class="pop-up-desc-desk">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+            <br>
+            <br>
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+            galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and
+            typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn
+            printer took a galley of type and scrambled it 1960s.
+          </p>
+        </div>
+        <div class="pop-up-desc">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Error et delectus quae at alias voluptatum
+            debitis pariatur eveniet eius quam!
+          </p>
+        </div>
+        <div class="side-pop">
+          <ul class="skill-list-desktop">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Bootstrap</li>
+          </ul>
+          <ul class="skill-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <div class="see-container">
+            <a href="">
+              <button>
+                <span>see live</span>
+                <img src="images/live-link-png.svg" alt="">
+              </button>
+            </a>
+            <a href="">
+              <button>
+                <span>see source</span>
+                <img style="width: 25px;" src="images/github-icon-png.png" alt="">
+              </button>
+            </a>
+          </div>
+        </div>
+      </div>
+  </section>
```

Comment:
- [ ] Great job, team, on making your modal's design look similar to the template and making the open and close buttons work as expected 👏. However, kindly ensure that it is not hard coded in the HTML. The modal should be generated dynamically with Javascript. When the modal opens, it should be prefilled with the information (image, title, description, etc.) that was displayed on the card where we clicked the see project button. 👌
So as stated in the requirements, you need to have a  JavaScript array to store all of the information for all projects and use a loop to go through the array of objects and display the information of a card with the modal whenever a see project button is clicked.  👍️

Please review [the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this task, which I have attached a screenshot below. :point_down: 
#### Task requirement :point_down: 
![Screenshot from 2023-06-23 13-24-13](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/89228743/50968055-652b-41d4-abcf-962875ce2cb4)